CNP_analysis.py

```diff
@@ -33,30 +32,39 @@
 #     nii = nii.__class__(data, nii.affine, nii.header).to_filename(out_file)
 #     return out_file, out_mask
 
-parser = argparse.ArgumentParser(
-    description='Perform analysis on CNP task data')
-parser.add_argument('-subject', '--subject', dest='subject',
-                    help='subject label', required=True)
-parser.add_argument('-prep_pipeline','--prep_pipeline',dest='prep_pipeline',
-                    help='preprocessing pipeline (fmriprep or feat)', required=True)
-args = parser.parse_args()
+def get_parser():
```

Comment:
Some improvements to the parser, wrapped inside a function (will help document later with automated tools)